test/integration/targets/connection_psrp/tests.yml

```diff
@@ -59,3 +59,42 @@
     assert:
       that:
       - async_out.stdout_lines == ["abc"]
+
+- name: Output unicode characters from Powershell using PSRP
+  raw: |
+    Write-Host '💩'
+  register: raw_unicode_output
+
+- name: Assert unicode output
+  assert:
+    that:
+    - raw_unicode_output is changed
+    - raw_unicode_output.rc == 0
+    - raw_unicode_output.stdout == '💩'
+    - raw_unicode_output.stderr == ''
+
+- name: Output unicode characters from Powershell using PSRP
+  win_command: |
```

Comment:
do `win_command: "powershell.exe -ExecutionPolicy ByPass -Command \"Write-Host '\U0001F4A9'\""`. Due to the PyYAML issue you can't put unicode characters that are a surrogate pair in YAML files.